includes/utilities.inc

```diff
@@ -505,13 +505,13 @@ function islandora_solr_truncate_field_display($display_values, $max_length, $ad
 
     $updated_display_values
       = "<span class='toggle-wrapper'>" .
-      t("<span>!value !separator !show_more</span>", array(
+      t("<span>!value !show_more</span>", array(
         '!separator' => $separator,
```

Comment:
Oh look at me be totally wrong. I missed the `variable_get` there. So I see what you mean. But do we want the **Show more** on its own line? Perhaps the separator can be removed and a `<br />` by hard coded in front of the **Show more** link?